enterprise/trackedusers/data_collector.go

```diff
@@ -0,0 +1,192 @@
+package trackedusers
+
+//go:generate mockgen -destination=./mocks/mock_data_collector.go -package=mockdatacollector github.com/rudderlabs/rudder-server/enterprise/trackedusers DataCollector
+
+import (
+	"context"
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	migrator "github.com/rudderlabs/rudder-server/services/sql-migrator"
+
+	obskit "github.com/rudderlabs/rudder-observability-kit/go/labels"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+
+	"github.com/rudderlabs/rudder-go-kit/logger"
+
+	"github.com/lib/pq"
+
+	"github.com/spaolacci/murmur3"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/segmentio/go-hll"
```

Comment:
[minor] 3rd party deps can be grouped together
```suggestion
	"github.com/lib/pq"
	"github.com/spaolacci/murmur3"
	"github.com/tidwall/gjson"
	"github.com/segmentio/go-hll"
```